ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1307,6 +1307,17 @@ tests:
     - ref: cucushift-helper-service-build-info
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-hypershift
+- as: azure-ipi-ovn-hypershift-p3-f21
+  cron: 32 0 24,1,8,15 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@azure-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.12'
```

Comment:
@LiangquanLi930 `TEST_IMPORTANCE: Critical` is added for such test.
why there is no it? because it is p3?